.github/workflows/autobuild.yml

```diff
@@ -151,14 +151,6 @@ jobs:
     runs-on:                        ${{ matrix.config.building_on_os }}
     steps:
 
-      # For Qt5 on Mac, we need to ensure SDK 10.15 is used, and not SDK 11.x
-      # This is done by selecting Xcode 11.7 instead of the latest default of 12.x
-      - name:                       Select Xcode version for Mac
-        if:                         ${{ matrix.config.target_os == 'macos' }}
-        uses:                       maxim-lobanov/setup-xcode@v1
-        with:
-          xcode-version: '11.7'
-
```

Comment:
Note that although the MacOS build appeared to succeed, it issued the following warnings:
```
Project WARNING: Qt has only been tested with version 10.15 of the platform SDK, you're using 12.0.
Project WARNING: This is an unsupported configuration. You may experience build issues, and by using
Project WARNING: the 12.0 SDK you are opting in to new features that Qt has not been prepared for.
Project WARNING: Please downgrade the SDK you use to build your app to version 10.15, or configure
Project WARNING: with CONFIG+=sdk_no_version_check when running qmake to silence this warning.
```
That is the reason why selecting the SDK version was added.

I expect it applies even more to the legacy build. Although it does not appear to do the above check, the warning probably still applies.

Reply:
Ok. So leave it?